pkgs/ffigen/lib/src/config_provider/config_types.dart

```diff
@@ -347,11 +347,11 @@ class CompilerOptsAuto {
 
   /// Extracts compiler options based on OS and config.
   List<String> extractCompilerOpts() {
-    if (Platform.isMacOS && macIncludeStdLib) {
-      return getCStandardLibraryHeadersForMac();
-    }
-
-    return [];
+    return [
+      if (Platform.isMacOS && macIncludeStdLib)
+        ...getCStandardLibraryHeadersForMac(),
+      if (Platform.isMacOS) '-Wno-nullability-completeness',
```

Comment:
Why only mac? All the configs use this flag on all platforms.

Reply:
I believe this flag does nothing on other OSes, it's an Xcode only flag.